ydb/docs/en/core/changelog-cli.md

```diff
@@ -1,8 +1,27 @@
 # {{ ydb-short-name }} CLI changelog
 
+## Version 2.23.0 {#2-23-0}
+
+Released on July 4, 2025. To update to version **2.23.0**, select the [Downloads](downloads/ydb-cli.md) section.
+
+### Features
+
+* Added the `{{ ydb-cli }} workload tpcc` command as a TPC-C benchmark.
+* Added the `{{ ydb-cli }} workload vector select` command to benchmark vector index performance and recall.
+* Added the `{{ ydb-cli }} tools infer csv` command to generate a `CREATE TABLE` SQL query from a CSV data file.
+
+### Improvements
+
+* Enhanced processing of special values (`null`, `/dev/null`, `stdout`, `cout`, `console`, `stderr`, and `cerr`) for the `--output` option in the `{{ ydb-cli }} workload * run` [commands](./reference/ydb-cli/commands/workload/index.md).
+* The `{{ ydb-cli }} workload` [commands](./reference/ydb-cli/commands/workload/index.md) now work with absolute paths.
+* Improvements in the `{{ ydb-cli }}` interactive mode:
+  * Improved inline hints.
+  * Added named expressions completion (i.e. table column names).
```

Comment:
Named expressions in YQL are a different feature that has nothing to do with column names: https://ydb.tech/docs/en/yql/reference/syntax/expressions#named-nodes